overlord/snapstate/snapstate.go

```diff
@@ -314,6 +316,26 @@ func refreshRetain(st *state.State) int {
 	return retain
 }
 
+// maxInhibitionTime returns the value of the maximum inhibition time
+func maxInhibitionTime(st *state.State) time.Duration {
```

Comment:
this and the constant should live in autorefresh.go. The function should be called ....Duration as it doesn't return a time or just maxInhinibition like the original const

Reply:
Done.